internal/runners/packages/list.go

```diff
@@ -162,7 +162,7 @@ func newFilteredRequirementsTable(requirements []*gqlModel.Requirement, filter s
 
 	rows := make([]packageRow, 0, len(requirements))
 	for _, req := range requirements {
-		if !strings.Contains(req.Requirement, filter) {
+		if !strings.Contains(strings.ToLower(req.Requirement), strings.ToLower(filter)) {
```

Comment:
I think this is a good case for `strings.EqualFold`

Reply:
I think we want to have a substring search though. If you look at the ticket you will see the user expected that searching for 'flask' and 'Flask' would result in matches for any package with the 'flask-' or 'Flask-' prefix.

Reply:
Ahh I see, okay makes sense 👍